src/Microsoft.DotNet.Build.Tasks.Packaging/src/GetApplicableAssetsFromPackages.cs

```diff
@@ -36,18 +36,18 @@ public class GetApplicableAssetsFromPackages : PackagingTask
         /// TargetMoniker to use when resolving assets.  EG: netcoreapp1.0, netstandard1.4
         /// </summary>
         [Required]
-        public string TargetMoniker { get; set; }
+        public string[] TargetMonikers { get; set; }
 
         /// <summary>
         /// runtime.json that defines the RID graph
         /// </summary>
         [Required]
         public string RuntimeFile { get; set; }
-        
+
         /// <summary>
         /// If specified will be used when resolving runtime assets, otherwise TargetMoniker will be used.
         /// </summary>
-        public string RuntimeTargetMoniker { get; set; }
+        public string[] RuntimeTargetMonikers { get; set; }
```

Comment:
These are the runtime monikers that decide which platform we are going to target ( eg. .NETCoreApp, Version=1.0/win7-x64). These wouldn't change the Helix targets.